src/js/edu-benefits/5490/config/form.js

```diff
@@ -69,9 +71,40 @@ const formConfig = {
       title: 'Education History',
       pages: {}
     },
-    schoolSelection: {
-      title: 'School Selection',
-      pages: {}
+    employmentHistory: {
+      title: 'Employment History',
+      pages: {
+        employmentHistory: {
+          title: 'Employment history',
+          path: 'employment-history',
+          uiSchema: {
+            nonMilitaryJobs: {
+              items: {
+                name: {
+                  'ui:title': 'Main job'
+                },
+                months: {
+                  'ui:title': 'Number of months worked'
+                },
+                licenseOrRating: {
+                  'ui:title': 'Licenses or rating'
+                }
+              },
+              'ui:options': {
+                itemName: 'Employment Period',
+                viewField: EmploymentPeriodView,
+                hideTitle: true
+              }
+            }
+          },
+          schema: {
+            type: 'object',
+            properties: {
+              nonMilitaryJobs: _.unset('items.properties.postMilitaryJob', nonMilitaryJobs)
```

Comment:
After talking with Alex, we realized it's from the 1990 form. This is using the same definition. It looks awkward to remove it on the backend, so I may leave it for now.